.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I'm a bit surprised by the addition of this file, I thought prettier is just used as it comes, much like `black` for python. 

Instead of `.prettierrc‎` please use `prettier.config.mjs` it aligns better with the other tooling configuration file names that we are already using. https://prettier.io/docs/configuration.

All of these options seem to be the default options, why configure them explicitly anyways?

I personally would vote for `"tabWitdth": 4`, I'm used to it, but also, the whole code base would require less formatting once we apply prettier to everything.